homeassistant/components/devolo_home_control/config_flow.py

```diff
@@ -0,0 +1,84 @@
+"""Config flow to configure the devolo home control integration."""
+import logging
+
+from devolo_home_control_api.mydevolo import Mydevolo
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+from homeassistant.core import callback
+
+from .const import (  # pylint:disable=unused-import
+    CONF_HOMECONTROL,
+    CONF_MYDEVOLO,
+    DEFAULT_MPRM,
+    DEFAULT_MYDEVOLO,
+    DOMAIN,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class DevoloHomeControlFlowHandler(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a devolo HomeControl config flow."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_CLOUD_PUSH
+
+    def __init__(self):
+        """Initialize devolo Home Control flow."""
+        self.data_schema = {
+            vol.Required(CONF_USERNAME): str,
+            vol.Required(CONF_PASSWORD): str,
+            vol.Required(CONF_MYDEVOLO, default=DEFAULT_MYDEVOLO): str,
+            vol.Required(CONF_HOMECONTROL, default=DEFAULT_MPRM): str,
+        }
+
+    async def async_step_user(self, user_input=None):
+        """Handle a flow initiated by the user."""
+        if user_input is None:
+            return self._show_form(user_input)
+        user = user_input[CONF_USERNAME]
```

Comment:
We need to check somewhere in the config flow that we don't create two entries with the same user. Should we allow more than one config entry?